src/synthesize.jl

```diff
@@ -160,39 +159,74 @@ function synthesize(atm::ModelAtmosphere, linelist, λs::AbstractRange; metallic
 end
 
 """
-    get_absolute_abundances(metallicity, abundances, solar_abundances, solar_relative)
+    construct_abundances(metallicity, abundances)
 
-Calculate ``n_X/n_\\mathrm{total}`` for each element X given abundances in either [``X``/H] or 
-``A(X)`` form.  See [`synthesize`](@ref) for a detailed description of the arguments. Returns a 
-vector indexed by atomic number.
+Returns a 92 element vector containing abundances in ``A(X)`` (``\\log(X/H) + 12``) format for 
+elements from hydrogen to uranium.
+
+# Arguments
+You can provide either or both of:
+- `metallicity` (default: 0), i.e. [metals/H] is the ``\\log_{10}`` solar-relative abundance of elements heavier 
+   than He. It is overriden by `abundances`.  
+- `abundances` is a `Dict` mapping atomic numbers or symbols to [``X``/H] abundances.  (Set 
+  `solar_relative=false` to use ``A(X)`` abundances instead.) These override `metallicity`.
```

Comment:
```suggestion
  `solar_relative=false` to use ``A(X)`` abundances instead.) These override `metallicity`.
  This is the only way to specify an abundance of He that is non-solar.
```
This is probably not necessary, but I think it could be useful. 

This is mostly motivated by the fact that I initially found the fact that metallicity doesn't affect He abundances to be somewhat non-obvious. After thinking about it, I realize that it makes a LOT of sense, but I think there might be some values in making things obvious for new users (especially for new students who might not have this intuition).

Reply:
This is actually not true.  You can override He  with the `abundances` dict regardless of whether with solar_relative is true or false, e.g. `construct_abundances(Dict("He"=>-1))` specified solar except with [He/H] = -1.

Reply:
I think there might be a miscommunication here - my suggestion was just to add some text emphasizing that if you want a non-solar abundance of Helium, you need to specify it in the `abundances` dict (as I currently understand it, metallicity has no effect on Helium). This edit has nothing to do with the value of the `solar_relative` keyword argument.

I was suggesting we make a change equivalent to adding the italicized text in the following snippet:

> **Arguments**
> You can provide either or both of:
> - `metallicity` (default: 0), i.e. [metals/H] is the $log_{10}$ solar-relative abundance of elements heavier 
>    than He. It is overriden by `abundances`, on a per-element basis.  
> - `abundances` is a `Dict` mapping atomic numbers or symbols to $[X/H]$ abundances.  (Set 
>   `solar_relative=false` to use ``A(X)`` abundances instead.) These override `metallicity`.
>    _This is the only way to specify an abundance of He that is non-solar._

As I mentioned before, this is only a minor suggestion to help people like me (i.e. new students), who might be a little surprised that the metallicity argument has no bearing on the value of Helium. I totally understand if you don't think this is a worthwhile addition to make (I'm personally not convinced that it's particularly useful).

Reply:
Ah, my bad.  Sorry for the misunderstanding--I read the suggested change too quickly.